config/webpack.config.dev.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const webpack = require('webpack')
+
+module.exports = {
+  devtool: '#source-map',
+  externals: ['cozy'],
+  module: {
+    loaders: [{
+      test: require.resolve('cozy-bar/dist/cozy-bar.js'),
+      loader: 'imports?css=./cozy-bar.css'
+    }]
+  },
+  plugins: [
+    new webpack.DefinePlugin({
+      __SERVER__: JSON.stringify('http://app.cozy.local'),
+      __STACK_ASSETS__: false
+    }),
+    new webpack.ProvidePlugin({
+      'cozy.client': 'cozy-client-js/dist/cozy-client.js',
+      'cozy.bar': 'cozy-bar/dist/cozy-bar.js'
```

Comment:
Nope, they're node modules, so in the `node_modules` folder. They're installed from the NPM registry as a _peerDependency_, but can be overriden with `cozy link` (or `yarn link`) command with local versions.

Webpack can properly handle them, as their subdir contents. The `ProvidePlugin` here acts like if you do `const cozy.client = require('cozy-client-js/dist/cozy-client.js')` at the top of each file which contains the variable `cozy.client`. And so on, Webpack access subdir contents of the module when you require a pattern like `<moduule_slug>/path/to/file`.